### PR TITLE
Discord domain verification thingy

### PR DESCRIPTION
**Pull request by keircn**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [/] The file is in the `domains` folder and is in the JSON format.
- [/] The file's name is all lowercased and alphanumeric. 
- [/] You have completed your website. 
- [/] The website is reachable.  
- [/] You're not using Vercel.  
- [/] The CNAME record doesn't contain `https://` or `/`.  
- [/] There is sufficient information at the `owner` field.  

## Website Link/Preview

image

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+keiran-is-a.dev
```
Comment (CuteDog5695) [inline CNAME:1]: Delete this file please

File: domains/_discord.keiran.is-a.dev [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (CuteDog5695) [inline domains/_discord.keiran.is-a.dev:1]: Replace .is-a.dev with .json

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Comment by Sxinar** [comment]
Hello, I want to use freeflarum now because netlify has frozen my account :/ I will use freeflarum cname: cname.freeflarum.com site link: sxinar.flarum.cloud

**Pull request merged**